Mathlib/Data/List/Basic.lean

```diff
@@ -1765,19 +1765,6 @@ theorem cons_get_drop_succ {l : List α} {n} :
 #align list.drop_take List.drop_take
 #align list.map_drop List.map_drop
 #align list.modify_nth_tail_eq_take_drop List.modifyNthTail_eq_take_drop
-
-@[simp]
-theorem modifyNth_nil (f : α → α) (n : ℕ) :
```

Comment:
Let me ask for the record: are these used anywhere? Is there a particular reason these are removed; is there an obvious alternative? Should these be deprecated? (I would not demand this, but would like to understand why no deprecation is warranted if not.)

Reply:
Oh, these have just moved upstream in the meantime.